src/isError.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable max-len */
+const availableCases = ['SNAKE' | 'KEBAB' | 'CAMEL' | 'PASCAL' | 'UPPER'];
```

Comment:
The elements of the 'availableCases' array need to be comma-separated strings. In the current format with the '|' operator, it's not creating an array of strings but rather a bitwise OR operation.